core/src/autogluon/core/trainer/abstract_trainer.py

```diff
@@ -1134,6 +1134,44 @@ def save(self):
         if self.low_memory:
             self.models = models
 
+    def compile_models(self, model_names='all', compiler_configs=None):
```

Comment:
Add `with_ancestors` argument, refer to persist_models for implementation

Reply:
Added the `with_ancestors` argument.